README.md

```diff
@@ -1,54 +1,49 @@
-Javascript + C++ BSON parser
-============================
+# BSON parser
 
-This BSON parser is primarily meant to be used with the `mongodb` node.js driver.
-However, wonderful tools such as `onejs` can package up a BSON parser that will work in the browser.
-The current build is located in the `browser_build/bson.js` file.
+If you don't yet know what BSON actually is, read [the spec](http://bsonspec.org).
+
+This package can be used to serialize JSON documents into the BSON format or the other way around. If you want to use it within the browser, give [browserify](https://github.com/substack/node-browserify) a try (it will help you add this package to your bundle). The current build is located in the `browser_build/bson.js` file.
 
 A simple example of how to use BSON in the browser:
 
 ```html
-<html>
-<head>
-  <script src="https://raw.github.com/mongodb/js-bson/master/browser_build/bson.js">
-  </script>
-</head>
-<body onload="start();">
+<script src="./browser_build/bson.js"></script>
+
 <script>
-  function start() {
-    var BSON = bson().BSON;
-    var Long = bson().Long;
+  function try() {
```

Comment:
Whoops! You're right. It's a preserved keyword. No idea why I've overlooked this... Fixed now!